configure_tls_protocol_version_and_ciphers.md

```diff
@@ -0,0 +1,18 @@
+# Configure TLS Protocol Version and Ciphers
+
+## SDK
+### How to configure TLS version
+Use `jdk.tls.client.protocols` system property to enforce TLS version. This property was introduced to JDK7 `7u95` and to JDK6 `6u121`.
```

Comment:
Suggest using 2 asterisks (bold) instead of backticks (commands) for things like jdk.tls.client.protocols and 7u95 that aren't commands.

Reply:
done